### ISSUE TITLE
Hosting on IIS

### ISSUE THREAD

**Issue by vvulovic** [aelassas/Wexflow#300]
Hello!

I need help with hosting this on IIS.

Can anybody explain to me which folder in IIS to target and do I need any additional configuration?

I have already tried a bunch of configuration stuff, but nothing works, it always says that the connection is blocked when I try to log in to the manager through web ui.

I only need the web UI to set it up.

ThanksS!

**Comment by janborup** [comment]
Hi @vvulovic 

It you have installed Wexflow as Windows Service, your /Backed is normally in a System-restricted area under ../ProgramFiles

Start by finding the /Backend folder, and copy this to a new area that can be accessed from ISS, normally C:\InetPub\wwwroot\WF-Backend\

Then make a simple new site to this area, but be sure NOT to use port 8000, because this is normally hosted by Wexflow, that could be port 80 or 8008.

On the site, be sure that "Index.html" is added in "Default document"
![image](https://user-images.githubusercontent.com/6866443/100556698-64acbe80-32a4-11eb-92bf-49edbb6fb621.png)

Now you should be ready to simply access the site
http://localhost:8008/

![image](https://user-images.githubusercontent.com/6866443/100556537-4e523300-32a3-11eb-8f96-50f4719c255c.png)

/Cheers

**Comment by vvulovic** [comment]
Hey @janborup ,

do I need to change anything in the js files?

Or any of the configs?

Thanks!

**Comment by janborup** [comment]
Nope..

If You dont use the standard port 8000 for Wexflow, You dont need to change anything.

/Cheers

**Comment by vvulovic** [comment]
@janborup 

This is what I get, we are using port 443 and 80

![Screenshot_2](https://user-images.githubusercontent.com/30145999/100607459-4201c080-330b-11eb-857e-0ba01f40e00c.png)

**Comment by janborup** [comment]
It looks like You are using some kind of domaine, and not localhost, then you got to have a totally other security setup.

I cannot help you There - sorry.

**Comment by vvulovic** [comment]
@janborup so, if not using domain, how would then I host it on iis?

**Comment by vvulovic** [comment]
@janborup alright, we think we know how to go around this. Basically, manager will run on / , so all the html files, and then, we will map everything that comes on hostname/wexlfow to localhost:8000/wexflow.

When we do this, we get this error:

details: "Error details are currently disabled.<br />To enable it, please set <strong>TraceConfiguration.DisplayErrorTraces</strong> to <strong>true</strong>.<br />For example by overriding your Bootstrapper's <strong>Configure</strong> method and calling<br/> <strong>environment.Tracing(enabled: false, displayErrorTraces: true);</strong>."
message: "Something went horribly, horribly wrong while servicing your request."
statusCode: 500

So, basically, we cannot do anything with it and I cannot find where to enable logs. Can you help me with this?

**Comment by janborup** [comment]
Hi

You start up by telling you want to host til Wexflow Client-Files in the
IIS, now you are talking about configure the Wexflow Rest-server to a
domain for the server, this is a completely different scenario.

You have to absolutely be clear about the difference.
- The Backend HTML files, has no server, but default configured to connect
to Wexflow port 8000
- The WexFlow installation on the server installed, is standard using
http://localhost - Port 8000

If you want to use your HTML files for a remote Wexfow server, you had to
change the Backend JavaScript files used for this, both with the
domain/server and the port, an configure and all firewall settings between
client and server should be configured.

*Tip ! *
Start by copy all /Backed files to your local computer, and setup the
connection for your Wexflow server, to be sure it works remote.

Best regards

*Jan Borup Coyle*


Den tir. 1. dec. 2020 kl. 12.07 skrev vvulovic <notifications@github.com>:
